src/docs/release/breaking-changes/target-platform-linux-windows.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Adding `linux` and `windows` to [`TargetPlatform`][] enum
+description: Two new values were added to the [`TargetPlatform`][] enum that could require additional cases in switch statements that switch on a [`TargetPlatform`][].
+---
+
+## Summary
+
+Two new values were added to the [`TargetPlatform`][] enum that could require additional cases in switch statements that switch on a [`TargetPlatform`][] and don't include a `default:` case.
+
+## Context
+
+Prior to this change, the [`TargetPlatform`][] enum only contained four values, and was defined like this:
+```dart
+enum TargetPlatform {
+  android,
+  fuchsia,
+  iOS,
+  macOS,
+}
+```
+
+A `switch` statement only needed to handle these cases, and desktop applications which wanted to run on Linux or Windows usually had a shim something like this in their main:
+
+```dart
+// Sets a platform override for desktop to avoid exceptions. See
+// https://flutter.dev/desktop#target-platform-override for more info.
+void _enablePlatformOverrideForDesktop() {
+  if (!kIsWeb && (Platform.isWindows || Platform.isLinux)) {
+    debugDefaultTargetPlatformOverride = TargetPlatform.fuchsia;
+  }
+}
+
+void main() {
+  _enablePlatformOverrideForDesktop();
+  runApp(MyApp());
+}
+```
+
+## Description of change
+
+The [`TargetPlatform`][] enum is now defined as:
+
+```dart
+enum TargetPlatform {
+  android,
+  fuchsia,
+  iOS,
+  linux, // new value
+  macOS,
+  windows, // new value
+}
+```
+
+And the shim in `main()` is no longer required.
+
+This can cause the Dart analyzer to give the [`missing_enum_constant_in_switch`][]
+warning for switch statements which don't include a `default` case. Writing a
+switch without a `default:` case is the recommended way to handle enums, since
+the analyzer can then help you find any cases which aren't handled.
```

Comment:
which => that

Guidance on what to use when:
https://www.quickanddirtytips.com/education/grammar/which-versus-that-0

Reply:
Fixed.  Thanks for the guidance.